### ISSUE TITLE
Remove sound effect on currency selector

### ISSUE THREAD

**Issue by goaaron** [pancakeswap/pancake-swap-interface#314]
Can you make it a feature to disable this? It can be annoying, especially on mobile.

**Comment by NickSdot** [comment]
Thanks for opening this issue! Was about to do the same right now.

Looks like a small issue, but it's indeed annoying to get every other audio stopped on mobile, for a complete useless and unnecessary sound effect.

Would appreciate to see the team working on this ASAP. Thanks in advance! 🙏

**Comment by brentkearney** [comment]
Yes please add a "mute" or disable sound effects feature. Super annoying.

**Comment by karelianpie** [comment]
It was done with https://github.com/pancakeswap/pancake-swap-interface/pull/334
